src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -23,13 +23,18 @@ public BlazorWebView()
 		public RootComponentsCollection RootComponents { get; }
 
 		/// <inheritdoc/>
-		public Func<ExternalLinkNavigationInfo, ExternalLinkNavigationPolicy>? OnExternalNavigationStarting { get; set; } 
+		public event EventHandler<ExternalLinkNavigationEventArgs>? OnExternalNavigationStarting;
```

Comment:
The event name shouldn't have `On` in it. The method that "invokes" it (the one below) should be `On[EventName]`.

Please confirm that this works in XAML when the BlazorWebView is defined there. The Control.Sample has a commented-out XAML snippet that you should be able to use, or of course start with a new project template (`dotnet new maui-blazor -o MyEventTestApp`).

Reply:
> Please confirm that this works in XAML when the BlazorWebView is defined there. The Control.Sample has a commented-out XAML snippet that you should be able to use, or of course start with a new project template (`dotnet new maui-blazor -o MyEventTestApp`).

This is how I tested the WPF XAML bits:
![image](https://user-images.githubusercontent.com/14852843/155403562-45e70970-054e-4cb4-a33c-d27d3629a92b.png)

Policy Configuration:
![image](https://user-images.githubusercontent.com/14852843/155403621-f1861bda-0097-450f-9bff-f4a80dc8b950.png)

Is that what you were looking for?

Reply:
Yes I think that looks right for the XAML.

But the event name still need to change and remove the `On` prefix. The event names are just things like "Pressed", "Loading" etc. and don't start with `On`.